packages/web/src/__generated__/.gitignore

```diff
@@ -0,0 +1,2 @@
+*
```

Comment:
You don't need the `.gitignore` inside the `__generated__`, ignore the entirely folder